pkg/config/config.go

```diff
@@ -108,6 +108,8 @@ type Registry struct {
 	// Auths are registry endpoint to auth config mapping. The registry endpoint must
 	// be a valid url with host specified.
 	Auths map[string]AuthConfig `toml:"auths" json:"auths"`
+	// DisableHTTP2Client disables http2 client operations (fetching/pulling)
+	DisableHTTP2Client bool `toml:"disable_http2_client" json:"disableHTTP2Client"`
```

Comment:
This comment should be updated to make it clear it just affects the image pull resolver.

Reply:
Also, think we can remove `Client` from the name, as there is no HTTP2 server component here.

Reply:
Done

Reply:
Oop, didnt see the client comment. Let me repush